homework-submission/week4hw/src/components/Form.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { observer, inject } from "mobx-react";
+
+@inject("todoListStore")
+@observer
+class Form extends React.Component {
+  render() {
+    const { onSubmitForm } = this.props.todoListStore;
+
+    return (
+      <form style={{ marginBottom: "20px" }} onSubmit={onSubmitForm}>
+        <input
+          type="text"
+          name="description"
+          placeholder="write the description"
+        />
+        <input type="text" name="deadline" placeholder="write the deadline" />
+        <input style={{ fontSize: "12px" }} type="submit" name="submit" />
```

Comment:
you forgot the value 🙃

Reply:
Since we don't have a default value so i didn't use it

Reply:
But  it dependes on the browser